Detectors/TPC/calibration/src/CalculatedEdx.cxx

```diff
@@ -53,7 +53,7 @@ void CalculatedEdx::setRefit(const unsigned int nHbfPerTf)
   mRefit = std::make_unique<o2::gpu::GPUO2InterfaceRefit>(mClusterIndex, &mTPCCorrMapsHelper, mFieldNominalGPUBz, mTPCTrackClIdxVecInput->data(), nHbfPerTf, mTPCRefitterShMap.data(), mTPCRefitterOccMap.data(), mTPCRefitterOccMap.size());
 }
 
-void CalculatedEdx::fillMissingClusters(int missingClusters[4], float minChargeTot, float minChargeMax, int method)
+void CalculatedEdx::fillMissingClusters(int missingClusters[4], float minChargeTot, float minChargeMax, int method, std::array<std::vector<float>, 5>& chargeMaxROC, std::array<std::vector<float>, 5>& mChargeMaxROC)
```

Comment:
I'm not sure I understand why `mChargeMaxROC` which has naming convention of a member is passed as local. Also, the naming of the passed variables `chargeMaxROC` and `mChargeMaxROC` is nearly identical.

Reply:
I accidently forgot to change it to `chargeMaxROC` and `chargeTotROC`. I just fixed it

Reply:
Perfect, thanks!